### PR TITLE
add WxWork v1.3

### PR DESCRIPTION
**Pull request by y00rb**
*If there’s a checkbox you can’t complete for any reason, that's okay, just explain in detail why you weren’t able to do so.*

After making all changes to the cask:

- [ ] `brew cask audit --download {{cask_file}}` is error-free.
- [ ] `brew cask style --fix {{cask_file}}` reports no offenses.
- [ ] The commit message includes the cask’s name and version.

Additionally, **if adding a new cask**:

- [ ] Named the cask according to the [token reference].
- [ ] `brew cask install {{cask_file}}` worked successfully.
- [ ] `brew cask uninstall {{cask_file}}` worked successfully.
- [ ] Checked there are no [open pull requests] for the same cask.
- [ ] Checked that the cask was not already refused in [closed issues].

[token reference]: https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md
[open pull requests]: https://github.com/caskroom/homebrew-cask/pulls
[closed issues]: https://github.com/caskroom/homebrew-cask/issues?q=is%3Aissue+is%3Aclosed

Wxwork is IM tool for wechat enterprise version.

### CODE REVIEW

File: Casks/wxwork.rb [hunk 1]
```diff
@@ -0,0 +1,10 @@
+cask 'wxwork' do
+  version '1.3.0'
+  sha256 '6a7415f0d60e2f898a42b97d814e83f5538df306f92f5dcaf672e8f3d6de0503'
+
+  url 'http://dldir1.qq.com/foxmail/work_weixin/WXWork_1.3.0.2300.dmg'
```
Comment (miccal) [inline Casks/wxwork.rb]: Please change this to `url "http://dldir1.qq.com/foxmail/work_weixin/WXWork_#{version}.dmg".

File: Casks/wxwork.rb [hunk 2]
```diff
@@ -0,0 +1,10 @@
+cask 'wxwork' do
+  version '1.3.0'
```
Comment (miccal) [inline Casks/wxwork.rb]: Please change this to `version '1.3.0.2300'`.

Comment (y00rb) [inline Casks/wxwork.rb]: fixed

File: Casks/wxwork.rb (hunk 1 above)
Comment (y00rb) [inline Casks/wxwork.rb]: fixed

**Comment by y00rb** [comment]
Changed, Thank you for review

**Comment by vitorgalvao** [comment]
@y00rb Next time, please fill the template.

**Pull request merged**